kalite/distributed/templates/distributed/partials/_footer.html

```diff
@@ -17,7 +17,7 @@
                         {{ channel_data.footer_text }}
                     </p>
                     <p class="footer-text">
-                        <img id="creative-commons-icon" src="{% static 'images/distributed/' True %}{{ channel_data.channel_license }}.png">
+                        <img id="creative-commons-icon" src="{% static 'images/distributed/' True %}{{ channel_data.channel_license }}.png" alt="Creative Commons logo">
```

Comment:
This is *not* the Creative Commons logo. Rather it is a specific license, in the case of our default Khan Academy content it is "Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States License" see Creative Commons for details.

This should also be wrapped in translation tags.